public/src/components/campaign_list.jsx

```diff
@@ -0,0 +1,49 @@
+
+import React, { Component } from 'react';
+import { Button, Table } from 'react-bootstrap';
+
+import TableRow from './campaign_table_row';
+
+export default class CampaignList extends Component {
+  componentDidMount() {
+    this.props.fetchAllCampaigns();
+  }
+
+  render() {
+    const { all_campaigns, account_info: { last_name, first_name } } = this.props;
+    return (
+      <div>
+        <h2>Welcome, {first_name} {last_name} to the Admin Campaigns Dashboard!</h2>
+        <Button bsSize="xsmall">Create New Phone Bank</Button>
```

Comment:
✔️ 
@perfectlynormalbeast So should i wait to merge this PR until he does? Or can I hook it up after?